docs/alternator/compatibility.md

```diff
@@ -183,6 +183,7 @@ feature's implementation is still subject to change and upgrades may not be
 possible if such a feature is used. For these reasons, experimental features
 are not recommended for mission-critical uses, and they need to be
 individually enabled with the "--experimental-features" configuration option.
+See [Enabling Experimental Features](#yaml_enabling_experimental_features) for details.
```

Comment:
I don't understand what is this `#yaml_enabling_experimental_features` thing work. It doesn't look like a normal link. Will it work on github? Or does it only work with Sphinx?

Reply:
#yaml_enabling_experimental_features is an anchor. Anchors allow us to link to a specific section on a page. First, you specify the anchor on a target page. Then you use it to create a link instead of using URL.
But you're right, this link is not working because it was placed in an include file for another Markdown file. I'll push and update with a workaround to make it work.

Reply:
I still don't understand why "enabling experimental features" is an internal link or an include file... Shouldn't it just be a different, stand-alone, page, and be a normal link?

Reply:
These are two different topics.

Experimental features:
I agree that there should be a separate page to cover the topic of experimental features, and I'm going to add such a page to the docs. Now it is covered in different subsections on different pages. I've opened an issue to make sure I remember about it: https://github.com/scylladb/scylladb/issues/12012
But at the moment, we need to link to a subsection.

External vs internal links:
- If you cross-reference another document in the same project, you need to use an internal link, or else it may break. The RST syntax is
` :doc:`Internal doc <relative path>`.
- If you cross-reference a section in a document in the same project, you need to a) create an anchor (it's called "target" in RST) and then use that anchor name to create a link. For example:
`.. _cql-cql-overview: - creates an anchor`.
`:ref:`CQL Overview <cql-cql-overview>`` - creates a link to the place in the document where you added that anchor.

- Links to other projects are external links, and there's their syntax:`
 `ScyllaDB Documentation <https://docs.scylladb.com/>`/_

Reply:
Please note that this is not an RST file, it's MD, I don't know if the same link syntax applies. In particular, it will be really good if https://github.com/scylladb/scylladb/blob/master/docs/alternator/compatibility.md continues to work.

In MD, can't we use good-old-HTML-like links, i.e., file name optionally followed by hash sign (`#`) and anchor name? Not assuming some sort of global anchor namespace?

Reply:
We can use the Markdown syntax, but in this case, it's not working because the target file [scylla-yaml.inc](https://github.com/scylladb/scylladb/blob/master/docs/operating-scylla/scylla-yaml.inc) follows the RST syntax.
I'm afraid I can't make it work, so I'll just add a regular URL. I'll replace it with the link to the Experimental Features pace, once we have it.